### ISSUE TITLE
Does eftest has support of global fixtures?

### ISSUE THREAD

**Issue by alekseysotnikov** [weavejester/eftest#78]
Like in `circleci.test` https://github.com/circleci/circleci.test#global-fixtures

**Comment by weavejester** [comment]
No, not currently. However the original author of fixtures [now thinks there are better alternatives](https://stuartsierra.com/2016/05/19/fixtures-as-caches) to them.

**Comment by frenchy64** [comment]
> How can I define global fixtures that are only run once for the entire test run? I just need an idea.

Try Leiningen's `:injections` keyword and enable it when you call eftest.

```
lein with-profile +test-injections eftest
```
```
(defproject ..
  :profiles {:test-injections {:injections [(prn "hello")]}}) 
```
